packages/lesswrong/server/fmCrosspost/crosspost.ts

```diff
@@ -71,23 +68,22 @@ const updateCrosspost = async (postId: string, draft: boolean, deletedDraft: boo
 
 export const handleCrosspostUpdate = async (document: DbPost, data: Partial<DbPost>) => {
   if (
-    (data.draft !== undefined || data.deletedDraft !== undefined || data.title !== undefined) &&
+    denormalizedFieldKeys.some((key) => data[key] !== undefined) &&
```

Comment:
Again, not a problem with this PR per se, but this doesn't really do what you want it to do. The data will contain everything in the form that was submitted, not everything changing. I'm 95% sure that document is right now the old state of the document, so you can add another condition which is `data[key] !== document[key]` and then you'll be more effectively preventing spurious updates.